.github/workflows/ci.yml

```diff
@@ -69,7 +69,7 @@ jobs:
 
     - name: Unit Test
       run: |
-        docker run --name chain --rm --net=host trufflesuite/ganache-cli --account="0x6aeeb7f09e757baa9d3935a042c3d0d46a2eda19e9b676283dce4eaf32e29dc9,100000000000000000000000" --account="0x7d51a817ee07c3f28581c47a5072142193337fdca4d7911e58c5af2d03895d1a,100000000000000000000000" --block-time 5 &
+        docker run --name chain --rm --net=host trufflesuite/ganache-cli -m "pistol kiwi shrug future ozone ostrich match remove crucial oblige cream critic" -e 100000 --block-time 5 &
```

Comment:
Instead of having the 100000 here which seems a big excessive we could also change the tests to just use 100 ETH. What do you think? Same would apply for the `README`.

Reply:
I agree, I only used 100000 Ether because it equals the 100000000000000000000000 Wei used before.